build/ci/wait_demo_pr_checks.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -xe
+set +o pipefail
+
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_tests_passed=false
+  gh_pr_checks_command=(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository)
+  export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_branch`
+  if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
+    sleep 5
+    while [ `"${gh_pr_checks_command[@]}" | grep -i pending | wc -l` -gt 0 ]; do
+      sleep 15
+    done
+    summary_counter=0
+    seconds_to_wait_for_summary_test=20
```

Comment:
`summary_counter` -> `seconds_waited_for_summary_test`
and to avoid mixing them up:
`seconds_to_wait_for_summary_test` -> `summary_test_timeout_seconds`?